manila/share/drivers/netapp/options.py

```diff
@@ -232,6 +232,11 @@
                default=3600,  # One Hour
                help='The maximum time in seconds to wait for a snapmirror '
                     'release when breaking snapmirror relationships.'),
+    cfg.StrOpt('netapp_snapmirror_schedule',
+               default='10min',  # 10 Mins
+               help='The name of the schedule either in mins or hours is '
+                    'used to update the SnapMirror relationship. Few valid '
```

Comment:
Is it the name of a *pre-configured* interval in netapp? Or is it *any* value with a time unit suffix? This comment doesn't make it clear which.
I guess it's just an interval, then I wouldn't say "name", but instead:

```suggestion
               help='An interval in either minutes or hours used used to '
                    'update the SnapMirror relationship. Few valid '
```

Reply:
I believe its name of pre-configured interval in netapp.

Reply:
Right https://docs.netapp.com/us-en/ontap-cli-98/job-schedule-cron-show.html#description, so we need to ensure, that it exists

I did not find a documentation about the cron schedules, that are available by default - but '10min' is definitely not everywhere pre-configured

Reply:
When bringing this upstream, setting the new config to 'hourly' to preserve existing behaviour makes sense, I think.
I see no need to make this different here downstream for us.

We would set '10min' in our helm-charts, then

Reply:
updated PR, changed to hourly